iceoryx2-ffi/cxx/src/config.cpp

```diff
@@ -11,13 +11,15 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
 #include "iox2/config.hpp"
+#include "iox/assertions_addendum.hpp"
 
 namespace iox2 {
 ConfigView::ConfigView(iox2_config_ptr ptr)
     : m_ptr { ptr } {
 }
 
 auto ConfigView::to_owned() const -> Config {
+    // IOX_TODO();
```

Comment:
Can this be removed?

Reply:
Nope, since this needs to still be implemented and in issue #264 is a point to remove all `IOX_TODO` before it can be closed. So it needs to stick around for a little longer.